lib/Connectable.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-lwhmkz.php, but found phpcs-scan-lWhmKZ.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).